tests/creation.test.ts

```diff
@@ -40,19 +37,12 @@ describe('Safe creation', () => {
     it('should fail if the signer is not connected to a provider', async () => {
       // Used to mock a signer without provider
       const mockedSigner = {}
-      const ethersSafeFactory = new EthersSafeFactory(
-        mockedSigner as Signer,
-        proxyFactoryAddress,
-        safeSingletonAddress
-      )
       await chai
         .expect(
-          ethersSafeFactory.createSafe({
-            owners,
-            threshold: -1
-          })
+          () =>
+            new EthersSafeFactory(mockedSigner as Signer, proxyFactoryAddress, safeSingletonAddress)
         )
-        .rejectedWith('Signer must be connected to a provider')
+        .throws('Signer must be connected to a provider')
```

Comment:
Is there a reason for using `throws` instead of `rejectedWith`? In case of errors raised by us, I'll use always the same for consinstency

Reply:
I have double checked it and I get the reason. It's because the constructor doesn't return a promise.